code401challenges/src/test/java/code401challenges/BinarySearchTest.java

```diff
@@ -0,0 +1,13 @@
+package code401challenges;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+public class BinarySearchTest {
+    @Test public void testBinaryMachine() {
+        int[] arr = {1, 2, 3, 4, 5, 6};
+        int key = 1;
+        int actualOutput = BinarySearch.BinaryMachine(arr, key);
+        System.out.println("actualOutput = " + actualOutput);

```

Comment:
For this to be an actual test, it needs to contain an assertion, not just a System.out.println. So in this case, that might look like:
`assertEquals("should find 1 at index 0", 0, actualOutput);`